config.yaml

```diff
@@ -1 +1,21 @@
-options: {}
+options:
+  from-resources:
+    default: False
+    description: Set to true if TimescaleDB is to be installed from resources.
+    type: boolean
+  apt-repository:
+    default: https://packagecloud.io/timescale/timescaledb/ubuntu/
+    description: |
+      The apt-repository to use for installing TimescaleDB.
+    type: string
+  apt-key:
+    default: https://packagecloud.io/timescale/timescaledb/gpgkey
+    description: |
+      The apt GPG key to use for installing TimescaleDB, in conjunction with
+      'apt-repository'. If it does not apply, configure to empty.
+    type: string
+  version:
+    default:
```

Comment:
what does an empty default mean here?

Reply:
It means the latest version is to be installed.